<html>
<body>
報告未在相應的塊中使用的
<b>catch</b> 參數。 該檢查不報告任何命名為 "ignore"
或 "ignored" 的 <b>catch</b> 參數。
<p><b>範例:</b></p>
<pre><code>
  try {
      def arr = new int[3]
      arr[5] = 5
  } catch(Exception <b>ex</b>) {
      println('Catching the exception')
  }
</code></pre>
<p>這裡的參數 <b>ex</b> 永遠不會在 <b>catch</b> 塊中使用。
<p>在套用快速修復後：</p>
<pre><code>
  try {
      def arr = new int[3]
      arr[5] = 5
  } catch(Exception <b>ignored</b>) {
      println('Catching the exception')
  }
</code></pre>
</body>
</html>
